src/components/Header/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Grid } from '@mui/material';
+import { styled } from '@mui/system';
+
+const StyledLogoPL = styled('img')`
+  height: 60px;
+`;
+const StyledFlag = styled('img')`
+  height: 32px;
+`;
+const StyledLine = styled('div')`
+  border-left: 3px solid ${({ theme }) => theme.palette.secondary.main};
+  height: 50px;
+`;
+
+export const Header = () => {
+  return (
+    <Box
+      position="fixed"
+      top={0}
+      left={0}
+      right={0}
+      marginLeft="auto"
+      marginRight="auto"
+      sx={{ backgroundColor: 'primary.light', width: '100%' }}
+      data-cy="header"
+    >
+      <Grid
+        container
+        direction="row"
+        justifyContent="space-between"
+        alignItems="center"
+        sx={{
+          px: 6,
+          height: 90,
+        }}
+        data-cy="header-grid"
+      >
+        <StyledLogoPL src="src/assets/ug_logo_pl.png" alt="blue UG logo"></StyledLogoPL>
+        <StyledLine />
+        <Grid item xs={7} />
```

Comment:
I think it would be better to get rid of this Grid element and put both logos into two divs with border-right and border-left respectively, it may make it easier to scale this header in the future

Reply:
ok